style.css

```diff
@@ -3,6 +3,53 @@
   padding: 0;
 }
 
+#overlay {
+  display: none;
```

Comment:
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.💯 🥇 

 - [ OPTIONAL] Nice implementing the design, however, it would be better if you add a little bit of blur to your overlay in order to match the Figma design and also make a good contrast 🎨 👨🏻‍🎨 





-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
|![18](https://user-images.githubusercontent.com/79058364/188521463-0e06a407-03b8-4e56-8ff6-ca03edbceacc.png) | ![image](https://user-images.githubusercontent.com/61790587/167505757-7e8c3eba-c166-4f42-842b-829d676cabd5.png) |